Mod1/Lessons/Week3/Classes.md

```diff
@@ -202,23 +184,21 @@ Unicorn fred = new Unicorn("Fred", "Silver");
 What we have just done is a very common pattern. We gave our constructor some arguments and we saved those arguments to **properties**. While this is a strong pattern, it is not a rule. For instance, you may want to set a property in your constructor that has a default value that isn't set using an argument:
 
 ```c#
-namespace Classes
+internal class Unicorn
```

Comment:
I'm thinking we should maybe not use "internal" here. When I was working on the access modifiers lessons it seemed like public and private were the only modifiers students need right now, and I'm hesitant to introduce another modifier unless there is a compelling reason.

Reply:
We talked and turns out visual studio defaults to internal. Moving forward for examples we are going to try to use only public and private, but not going to go back and change. Moving forwards!